lib/KCard/index.vue

```diff
@@ -1,20 +1,59 @@
 <template>
 
-  <BaseCard
-    :to="to"
-    :title="title"
-    :headingLevel="headingLevel"
-    :titleLines="titleLines"
-    :class="['k-card', rootClass, thumbnailAlignClass]"
-    :headingStyles="headingStyles"
+  <!-- see trackInputModality  for [data-focus=true] -->
+  <!--
+    do not use @onmouseup but rather @click
+    to allow for @click.stop on buttons and similar
+    rendered within a card via its slots -->
+  <li
+    tabindex="0"
+    data-focus="true"
+    :class="['k-card', $computedClass(coreOutlineFocus)]"
+    @focus="onFocus"
+    @mouseenter="onHover"
+    @mousedown="onMouseDown"
+    @click="onClick"
+    @keyup.enter="onEnter"
   >
-    <template v-if="$slots.title" #title>
-      <!-- @slot Optional slot section containing the title contents, should not contain a heading element. -->
-      <slot name="title"></slot>
-    </template>
-    <template #default>
+    <div
+      :class="['card-area', layoutClass, thumbnailAlignClass]"
+      :style="{ backgroundColor: $themeTokens.surface }"
+    >
+      <component
+        :is="headingElement"
+        v-if="title || $slots.title"
+        class="heading"
+        :style="{ color: $themeTokens.text }"
+      >
+        <!--
+          Prevent router-link click event by setting empty event=""
```

Comment:
I know this was moved from BaseCard, but I dont understand well what empty event this refers to, perhaps the comment is about why the router-link has tabindex="-1"?

Reply:
Oh I think previously there was `event=""` applied to `router-link` and this comment was about that. I must have lost it during the refactor. I will investigate if it's still needed. Good catch!

Reply:
Yup, I need to return it to keep the feature for making card's text selectable. Done, also tried to improve the comment.